supporting_scripts/course-scripts/quick-course-setup/README.md

```diff
@@ -0,0 +1,133 @@
+# Artemis Course Setup Scripts
+
+This project contains Python scripts that automate the setup and management of courses and users in an Artemis instance. The scripts help you create a course, generate users, enroll them into courses, and manage programming exercises, including participation and commits. The project is useful for setting up test environments or quickly initializing a course with a large number of students.
+
+## Setup
+
+### 1. Install Python and the Python Plugin for IntelliJ
+
+- Ensure that Python (preferably version 3.9) is installed on your system.
```

Comment:
Why do we not directly supply the command to check the current version? Not everybody might be fluent in python and know that you have to check with `python3` instead of `python`

```suggestion
- Ensure that Python (preferably version 3.9) is installed on your system.
   ```shell
   python3 --version
   ```
```